backend/src/data_access_layer/point_repository.js

```diff
@@ -1,24 +1,26 @@
 const admin = require("firebase-admin")
 
-module.exports = function ({ }) {
+module.exports = function () {
 
      return {
           /**
            * @param {Map<String, Any>[]} callback
            */
-          getAllPoints: function (callback) {
-               // db.firestore  === undifined
-               // console.log(db.firestore)
-               // console.log(typeof db.firestore)
-
-               const docRef = admin.firestore().collection('WallE').doc('test');
-               docRef.set({
-                    first: 'Adasdaa',
-                    last: 'Lovelace',
-                    born: 2000
-               });
-
-          },
+          getAllPoints:  function (callback) {
+               const docRef = admin.firestore().collection('maps');
+               docRef.get().then((docSnap)=> {
+                    if(docSnap.empty){
+                         callback('Doc does not exists', null)
```

Comment:
good error messige, but remember we are creating an API, so we send error codes I would suggest 'NoExistingMap'